lemur/logs/service.py

```diff
@@ -28,7 +28,8 @@ def create(user, type, certificate=None):
         "function": "lemur-audit",
         "action": type,
         "user": user.email,
-        "certificate": certificate.name
+        "certificate": certificate.name,
+        "message": f"[lemur-audit] action: {type}, user: {user.email}, certificate: {certificate.name}."
     }
     # format before August 2021: f"[lemur-audit] action: {type}, user: {user.email}, certificate: {certificate.name}."
```

Comment:
I'm not sure if it's helpful to leave this comment or not, as I reused the old format for this new attribute.